docs/src/components/pricing/PricingTable.tsx

```diff
@@ -601,6 +603,33 @@ const rowHeaders: Record<string, React.ReactNode> = {
   'charts/filter-interaction': <ColumnHead label="Row filtering" />,
   'charts/selection-interaction': <ColumnHead label="Range selection" />,
   'tree-view/tree-view': <ColumnHead label="Tree View" href="/x/react-tree-view/" />,
+  'tree-view/item-selection': (
+    <ColumnHead label="Item selection" href="/x/react-tree-view/rich-tree-view/selection/" />
```

Comment:
Maybe clearer

```suggestion
    <ColumnHead label="Single selection" href="/x/react-tree-view/rich-tree-view/selection/" />
```

Reply:
Should we update the grid equivalent feature?

Reply:
In my view, the data grid, needs to differentiate between row and cell selection, but it stills uses the same terminology as in the docs. I would see the pricing page match what the docs uses. So I think "Should we update the grid equivalent feature?" is either 

<img width="260" alt="SCR-20240610-upme" src="https://github.com/mui/material-ui/assets/3165635/9af31677-d0a1-423a-821a-7fb82629870f">

should stay like this, or become:

<img width="206" alt="SCR-20240610-uprx" src="https://github.com/mui/material-ui/assets/3165635/8a4a6ea4-b5b2-4cac-9136-c41d1315717e">

I personally feel that it works well in the docs.